crates/walrus-service/storage_openapi.yaml

```diff
@@ -1063,6 +1070,13 @@ components:
                 description: The number of events that have been persisted.
                 minimum: 0
           description: The event progress of the storage node.
+        latestCheckpointSequenceNumber:
+          type:
+          - integer
+          - 'null'
+          format: int64
+          description: The latest checkpoint sequence number processed by the node.
```

Comment:
This should be "Sequence number of the latest checkpoint downloaded by the node."